MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -37,6 +39,11 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		tableModel = cellControllers
 	}
 
+	func display(_ viewModel: FeedErrorViewModel) {
+		guard let errorMessage = viewModel.errorMessage else { return self.errorView.hideMessage() }
+		self.errorView.show(message: errorMessage)
```

Comment:
Suggestion: 

An `if/else` statement is usually easier to read/follow in such cases:

"If there's a message to show, show it. Else, hide the existing one."

```suggestion
		if let errorMessage = viewModel.errorMessage {
			self.errorView.show(message: errorMessage)
		} else {
			self.errorView.hideMessage()}
		}
```

Instead of "guard against not having a message. Else, hide the existing one. Show the message if it passed the guard statement."